src/main/java/de/javagl/jsonmodelgen/json/schema/v4/codemodel/CodeModelDocs.java

```diff
@@ -171,6 +171,12 @@ static String createJavaDoc(List<String> lines)
         {
             descriptionLines.add("Valid values: " + 
                 propertySchema.getEnumStrings());
+        } else {
+            List<String> nonObjectEnumStrings = SchemaCodeUtils.determineEnumStringsFromAnyOf(propertySchema);
+            if (nonObjectEnumStrings != null && !nonObjectEnumStrings.isEmpty()) {
+                descriptionLines.add("Valid values: " +
+                        nonObjectEnumStrings);
+            }

```

Comment:
This chunk adds the "Valid values" bit when those are defined as part of `anyOf` (they were already checked on the verification step, but not added to the docs, which might be confusing)